renderdoc/driver/vulkan/wrappers/vk_shader_funcs.cpp

```diff
@@ -1464,14 +1481,37 @@ VkResult WrappedVulkan::vkCreateRayTracingPipelinesKHR(
 {
   VkResult ret;
 
-  // to be extra sure just in case the driver doesn't, set pipelines to VK_NULL_HANDLE first.
+  VkRayTracingPipelineCreateInfoKHR *unwrappedCreateInfos =
+      UnwrapInfos(m_State, pCreateInfos, createInfoCount);
+
   for(uint32_t i = 0; i < createInfoCount; i++)
+  {
+    // to be extra sure just in case the driver doesn't, set pipelines to VK_NULL_HANDLE first.
     pPipelines[i] = VK_NULL_HANDLE;
 
+    // Patch in capture/replay creation flags
+    VkBaseOutStructure *nextStruct = (VkBaseOutStructure *)unwrappedCreateInfos[i].pNext;
```

Comment:
This can be done more simply with 

```
VkPipelineCreateFlags2CreateInfoKHR *flagsInfo =
        (VkPipelineCreateFlags2CreateInfoKHR *)FindNextStruct(
            &unwrappedCreateInfos[i], VK_STRUCTURE_TYPE_PIPELINE_CREATE_FLAGS_2_CREATE_INFO_KHR);
```